site/docs/community.md

```diff
@@ -60,5 +60,9 @@ Iceberg has three mailing lists:
     - [Subscribe](mailto:commits-subscribe@iceberg.apache.org)
     - [Unsubscribe](mailto:commits-unsubscribe@iceberg.apache.org)
     - [Archive](https://lists.apache.org/list.html?commits@iceberg.apache.org)
-* **Private**: <private@iceberg.apache.org> -- private PMC list for committer nominations
+* **Issues**: <issues@iceberg.apache.org> -- Github issue tracking
+    - [Subscribe](mailto:issues-subscribe@iceberg.apache.org)
+    - [Unsubscribe](mailto:issues-unsubscribe@iceberg.apache.org)
+    - [Archive](https://lists.apache.org/list.html?issues@iceberg.apache.org)
+* **Private**: <private@iceberg.apache.org> -- private list for the PMC to discuss issues related to the health of the project
```

Comment:
Could you revert the change to this description? I don't think it's accurate.

We usually want discussion about any subject to be on the dev list. The only time we want to use the private list is when the subject matter is sensitive, like committer nominations, how to interact with specific companies to enforce trademarks, or security issues. I think that the health of the project should be discussed openly.

Reply:
I'm happy to update to 'sensitive issues,' but as an ASF Member and Incubator PMC member and mentor, this is the description I would use and I think it covers what you described.

Reply:
I think "sensitive issues" works. Thanks!